<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SATendencyChartDataSelectDialog</class>
 <widget class="QDialog" name="SATendencyChartDataSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tendency Chart Data Select</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioButton_follow">
     <property name="text">
      <string>follow define curve x values</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>choose a curve to follow</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView_curs"/>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_selDef">
     <property name="text">
      <string>define x value</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>start:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_start">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Growth value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_added">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_selDef</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_start</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_selDef</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_added</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_selDef</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_selDef</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_follow</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_follow</sender>
   <signal>toggled(bool)</signal>
   <receiver>listView_curs</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SATendencyChartDataSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
